robustus/robustus.py

```diff
@@ -553,7 +553,7 @@ def install(self, args):
                 active_branch = re.search('origin/\w*', msg).group(0)
                 # http://stackoverflow.com/questions/6657690/python-getoutput-equivalent-in-subprocess
             except Exception as err:
-                active_branch = '<Could not find active branch - %s: %s>' % (err.__class__.__name__, err.message)
+                active_branch = '<Could not find active branch - %s: %s>. Branch out of date. Try running git pull' % (err.__class__.__name__, err.message)
```

Comment:
Please change to "Branch might be out of date". I just think there might be other reasons to get this error and people will be confused again.